docs/host/host.md

```diff
@@ -251,3 +251,32 @@ When Ksmtuned starts, initialize `pages_to_scan` in KSM to 1000 (**Minimum Pages
 KSM starts when the available memory falls below the **Threshold Coefficient**. If it detects that it is running, `pages_to_scan` increments by 300 (**Boost**) every minute until it reaches 5000 (**Maximum Pages**).
 
 KSM will stop when the available memory is above the **Threshold Coefficient**. If it detects that it is stopped, `pages_to_scan` decrements by 100 (**Decay**) every minute until it reaches 1000 (**Minimum Pages**).
+
+## NTP Configuration
+
+Time synchronization is important for the distributed cluster architecture, so there is a simple way to configure the NTP servers on Harvester.
+
+In the previous version, we only support NTP configuration when [installation](https://docs.harvesterhci.io/v1.2/install/harvester-configuration#osntp_servers). If you want to change the NTP configuration, manually update the configuration file on each node.
+
+In v1.2.0, Harvester supports NTP configuration on Harvester/Settings. You can configure NTP servers anytime for the whole Harvester cluster. The NTP configuration will be applied to all nodes in the cluster.
+
+![](/img/v1.3/host/harvester-ntp-settings.png)
+
+You could setup multiple NTP Servers like the following figure.
+
+![](/img/v1.3/host/harvester-ntp-settings-multiple.png)
+
+You can check the configuration result with CRD `node.harvesterhci.io/ntp-service`
+- `ntpSyncStatus`: synced or unsynced
+- `currentNtpServers`: current NTP servers
+
+```
+$ kubectl get nodes harvester-node-0 -o yaml |yq -e '.metadata.annotations.["node.harvesterhci.io/ntp-service"]'
+{"ntpSyncStatus":"synced","currentNtpServers":"0.suse.pool.ntp.org 1.suse.pool.ntp.org"}
+```
+
+There are points you need to know: 
+1. Do not directly modify the NTP config on the host. Node-manager will keep the consistency between the configuration on Harvester/Settings and the configuration on the host.
+2. Suppose your system is upgraded from the previous version. In that case, you need to manually configure your NTP servers on Harvester/Settings first because Harvester does not know your last NTP configuration if there are conflicts. So, for the upgrade case, your ntp-server will be empty, like the following figure.
```

Comment:
```suggestion
2. Suppose your system is upgraded from a previous version. In that case, you need to manually configure your NTP servers on Harvester/Settings first because Harvester does not know your last NTP configuration if there are conflicts. So, for the upgrade case, your ntp-server will be empty, like the following figure.
```